spec/services/issue_card_cloner_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'rails_helper'
+
+RSpec.describe IssueCardCloner do
+  describe 'GitHub API best practices' do
+    it 'should pause at least one second after POST requests' do
+      # Mock all dependencies
+      labels = [{name: 'Big Problem'}, {name: 'Lotta Work'}]
+      base_issue = double(:base_issue, title: 'Generic Issue Title', labels: labels, body: 'Super important task')
+      cloned_issue = double(:cloned_issue, id: 1)
+      client = double(:client, issue: base_issue, create_issue: cloned_issue, create_project_card: nil)
+      repo = double(:repo, full_name: 'SomeGithubUser/some-repo')
+      template_card = double(:template_card, content_url: "https://api.github.com/repos/example-org/example-repo/issues/1")
+      cloned_column = double(:cloned_column, id: 1)
+
+      time = Benchmark.measure {
```

Comment:
never used this, good find!

Reply:
It was a lot easier to implement than I thought!